### PR TITLE
[11.0] [ADD] project_task_default_tags

### PR DESCRIPTION
**Pull request by angelmoya**
This module add project tags on project, and set as default on new project tasks.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Use singular in module name.

File: project_task_default_tags/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```
Comment (tarteo) [inline project_task_default_tags/README.rst]: Can you use the new splitted template for this. https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

File: project_task_default_tags/views/project_view.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+    <record id="edit_project" model="ir.ui.view">
+        <field name="name">project.project.form.inherit</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project"/>
+        <field name="arch" type="xml">
+          <field name="sequence" position="after">
+            <field name="tag_ids" widget="many2many_tags" options="{'no_create_edit': True}"/>
+        </field>
```
Comment (tarteo) [inline project_task_default_tags/views/project_view.xml]: Please, fix the indentation here.

File: project_task_default_tags/tests/test_project_task_default_tags.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            'tag_ids': [(6, 0, self.tag_B.ids)]
+        })
+        self.task = self.env['project.task'].with_context({
+            'default_project_id': self.project_A.id
+        }).create({
+            'name': 'Task Test',
+        })
+
+    def test_new_task(self):
+        self.assertEqual(self.task.tag_ids, self.project_A.tag_ids)
```
Comment (tarteo) [inline project_task_default_tags/tests/test_project_task_default_tags.py]: Can you put `self.task.project_id = self.project_A` here to make sure in this test project_A is certainly the project of the task.

**Review by tarteo — CHANGES_REQUESTED** [review]

File: project_task_default_tags/tests/test_project_task_default_tags.py (hunk 3 above)
Comment (angelmoya) [reply project_task_default_tags/tests/test_project_task_default_tags.py]: There is not needed because I created task with project_id = project_A.

**Comment by angelmoya** [comment]
@pedrobaeza @tarteo 

Done, thanks

File: project_task_default_tag/models/project.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 PESOL (http://pesol.es)
+#                Angel Moya (angel.moya@pesol.es)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields, models, api
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    tag_ids = fields.Many2many(
```
Comment (dreispt) [inline project_task_default_tag/models/project.py:10]: IMO this should be named `default_tag_ids`.
The `tag_ids` fields should be used for a feature adding Projects level Tags.

**Pull request closed without merging**